packages/app/src/files-manager/components/files-summary-item/files-summary-item.test.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import "@testing-library/jest-dom"
+import React, { ComponentProps } from "react"
+import FilesSummaryItem from "App/files-manager/components/files-summary-item/files-summary-item.component"
+import { renderWithThemeAndIntl } from "Renderer/utils/render-with-theme-and-intl"
+import { Type } from "Renderer/components/core/icon/icon.config"
+import { FilesSummaryItemTestIds } from "App/files-manager/components/files-summary-item/files-summary-item-test-ids.enum"
+
+const defaultProps: ComponentProps<typeof FilesSummaryItem> = {
+  filesType: "System",
+  color: "#DFEFDE",
+  icon: Type.MuditaLogo,
+  occupiedMemory: 1024,
+}
+
+const render = () => {
+  return renderWithThemeAndIntl(<FilesSummaryItem {...defaultProps} />)
+}
+test("FilesSummaryItem should render properly", () => {
```

Comment:
a `describe` is missing. In my opinion, a `describe` as the root or roots of the tests is useful. Tests then are nicely grouped.
![Zrzut ekranu 2022-01-25 o 07 27 47](https://user-images.githubusercontent.com/17147149/150922911-e92c29b9-0d34-4b34-97f7-28e884789b84.png)

Reply:
Added in [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)